brainzutils/musicbrainz_db/release_group.py

```diff
@@ -0,0 +1,162 @@
+
+        if 'artists' in includes:
+            query = query.\
+                options(joinedload("artist_credit")).\
+                options(joinedload("artist_credit.artists")).\
+                options(joinedload("artist_credit.artists.artist"))
+
+        release_groups = get_entities_by_gids(
+            query=query,
+            entity_type='release_group',
+            mbids=mbids,
+        )
+        release_group_ids = [release_group.id for release_group in release_groups.values()]
+
+        if 'artists' in includes:
+            for release_group in release_groups.values():
+                artist_credit_names = release_group.artist_credit.artists
+                includes_data[release_group.id]['artist-credit-names'] = artist_credit_names
+                includes_data[release_group.id]['artist-credit-phrase'] = release_group.artist_credit.name
+
+        if 'releases' in includes:
+            query = db.query(models.Release).filter(getattr(models.Release, "release_group_id").in_(release_group_ids))
+            for release in query:
+                includes_data[release.release_group_id].setdefault('releases', []).append(release)
+
+        if 'release-group-rels' in includes:
+            get_relationship_info(
+                db=db,
+                target_type='release_group',
+                source_type='release_group',
+                source_entity_ids=release_group_ids,
+                includes_data=includes_data,
+            )
+
+        if 'url-rels' in includes:
+            get_relationship_info(
+                db=db,
+                target_type='url',
+                source_type='release_group',
+                source_entity_ids=release_group_ids,
+                includes_data=includes_data,
+            )
+
+        if 'work-rels' in includes:
+            get_relationship_info(
+                db=db,
+                target_type='work',
+                source_type='release_group',
+                source_entity_ids=release_group_ids,
+                includes_data=includes_data,
+            )
+
+        if 'tags' in includes:
+            release_group_tags = get_tags(
+                db=db,
+                entity_model=models.ReleaseGroup,
+                tag_model=models.ReleaseGroupTag,
+                entity_ids=release_group_ids,
+            )
+            for release_group_id, tags in release_group_tags:
+                includes_data[release_group_id]['tags'] = tags
+
+        for release_group in release_groups.values():
+            includes_data[release_group.id]['meta'] = release_group.meta
+            includes_data[release_group.id]['type'] = release_group.type
+        release_groups = {str(mbid): serialize_release_groups(release_groups[mbid], includes_data[release_groups[mbid].id])
+                          for mbid in mbids}
+        return release_groups
+
+
+def browse_release_groups(artist_id, release_types=None, limit=None, offset=None):
```

Comment:
Lets change the function name to `get_release_groups_for_artist()`. What do you think? The name of the function doesn't make it obvious on what it is supposed to do. Now that this function is part of a library, I think it makes more sense that the functions are easily understandable.